src/wp-includes/fonts/class-wp-font-family.php

```diff
@@ -0,0 +1,615 @@
+
+	/**
+	 * Gets the post for a font family.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return WP_Post|null The post for this font family object or
+	 *                      null if the post does not exist.
+	 */
+	public function get_font_post() {
+		$args = array(
+			'post_type'      => 'wp_font_family',
+			'post_name'      => $this->data['slug'],
+			'name'           => $this->data['slug'],
+			'posts_per_page' => 1,
+		);
+
+		$posts_query = new WP_Query( $args );
+
+		if ( $posts_query->have_posts() ) {
+			return $posts_query->posts[0];
+		}
+
+		return null;
+	}
+
+	/**
+	 * Gets the data for this object from the database and
+	 * sets it to the data property.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return WP_Post|null The post for this font family object or
+	 *                      null if the post does not exist.
+	 */
+	private function get_data_from_post() {
+		$post = $this->get_font_post();
+		if ( $post ) {
+			$this->data = json_decode( $post->post_content, true );
+			return $post;
+		}
+
+		return null;
+	}
+
+	/**
+	 * Creates a post for a font family.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return int|WP_Error Post ID if the post was created, WP_Error otherwise.
+	 */
+	private function create_font_post() {
+		$post = array(
+			'post_title'   => $this->data['name'],
+			'post_name'    => $this->data['slug'],
+			'post_type'    => 'wp_font_family',
+			'post_content' => $this->get_data_as_json(),
+			'post_status'  => 'publish',
+		);
+
+		$post_id = wp_insert_post( $post );
+		if ( 0 === $post_id || is_wp_error( $post_id ) ) {
+			return new WP_Error(
+				'font_post_creation_failed',
+				__( 'Font post creation failed' )
+			);
+		}
+
+		return $post_id;
+	}
+
+	/**
+	 * Gets the font faces that are in both the existing and incoming font families.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $existing The existing font faces.
+	 * @param array $incoming The incoming font faces.
+	 * @return array The font faces that are in both the existing and incoming font families.
+	 */
+	private function get_intersecting_font_faces( $existing, $incoming ) {
+		$intersecting = array();
+		foreach ( $existing as $existing_face ) {
+			foreach ( $incoming as $incoming_face ) {
+				if ( $incoming_face['fontStyle'] === $existing_face['fontStyle'] &&
+					$incoming_face['fontWeight'] === $existing_face['fontWeight'] &&
+					$incoming_face['src'] !== $existing_face['src'] ) {
+					$intersecting[] = $existing_face;
+				}
+			}
+		}
+		return $intersecting;
+	}
+
+	/**
+	 * Updates a post for a font family.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param WP_Post $post The post to update.
+	 * @return int|WP_Error Post ID if the update was successful, WP_Error otherwise.
+	 */
+	private function update_font_post( $post ) {
+		$post_font_data = json_decode( $post->post_content, true );
+		$new_data       = WP_Font_Family_Utils::merge_fonts_data( $post_font_data, $this->data );
+		if ( isset( $post_font_data['fontFace'] ) && ! empty( $post_font_data['fontFace'] ) ) {
```

Comment:
```suggestion
		if ( isset( $post_font_data['fontFace'] ) && ! empty( $post_font_data['fontFace'] ) ) {
```
Looks like we can lose the `isset()` from here, as well as on line 534 just below this.

Reply:
Resolved by https://github.com/WordPress/wordpress-develop/pull/5285/commits/feb11999f753c808a18dd44a9561f74e91bf009a in both places ✅